library/core/src/main/java/com/google/android/exoplayer2/text/ssa/SsaDecoder.java

```diff
@@ -50,6 +53,9 @@
   private int formatEndIndex;
   private int formatTextIndex;
 
+  private int playResX;
+  private int playResY;
```

Comment:
It's probably safer to explicitly initialise these to an invalid value to clearly indicate 'unset', because 0 seems like a potentially genuine value we could see in a subtitle file? And then also update the comparison below on L216.

We have `C.LENGTH_UNSET` which I think would work well.
https://github.com/google/ExoPlayer/blob/release-v2/library/core/src/main/java/com/google/android/exoplayer2/C.java